build_scripts/build_deps.sh

```diff
@@ -34,6 +34,7 @@ echo ${EXTRA_PROTOBUF_FLAGS}
 echo ${OPENCV_TOOLCHAIN_FILE}
 echo ${EXTRA_FLAC_FLAGS}
 echo ${EXTRA_LIBSND_FLAGS}
+echo ${EXTRA_LIBTAR_FLAGS}
```

Comment:
I don't think we need that here. Do you know about any, platform specific flags that we need to pass to libtar?

Reply:
Nope, just added it because it's also for libsnd. Should I remove it?

Reply:
Yes, please. It is not needed here.

Reply:
done